src/constants/constants.ts

```diff
@@ -45,6 +45,7 @@ export const azureDatabase = 'Azure';
 export const defaultPortNumber = 1433;
 export const sqlAuthentication = 'SqlLogin';
 export const defaultDatabase = 'master';
+export const defaultDatabaseName = '<default>';
```

Comment:
Shouldn't this be localized? It looks like we don't do this for ADS but I think that's wrong since this should just be a display value.